src/index.ts

```diff
@@ -717,8 +717,24 @@ export class GalleryComponent extends BaseComponent {
     this._range = Maths.clamp(norm, 0.05, 1);
   }
 
-  private _setThumbMultiSelected(thumb: Thumb, selected: boolean): void {
-    $.observable(thumb).setProperty("multiSelected", selected);
+
+  // Update the DOM when the multiSelected state changes
+  private _updateThumbHtmlMultiSelected(thumbIndex: number, multiSelected: boolean): void {

```

Comment:
ugly but without data-link from jsviews which needs thew teamplating engine part of it, I couldn't think of a better way of dynamically updating the checkbox html. The MultiselectEnabled box clicks are triggered above the level of the gallerycomponent in examples/index.html so that;'s how that is able to define whether the select button is shown without the jsviews  or something like this. i didn't want to pull the multiselect logic out of GalleryComponent and do something similar as that would be a breaking change